CHANGELOG

```diff
@@ -1,5 +1,8 @@
 knife-vsphere changelog
 ----------------------
+2.0.2   bseeto  - Add snapshot creation timestamp to snapshot listing.
```

Comment:
Normally we use the GitHub account of the contributor as the name. Completely up to you on if/how your name appears in the changelog (and thanks for updating it), but thought I'd mention it.

Reply:
Yep, I can change this!